### PR TITLE
Update @nextcloud/vue to 6.0.0-beta.1

### PR DESCRIPTION
**Pull request by PVince81**
and also vue-template-compiler to 2.7.8
and vue to 2.7.8

- [ ] how to test ?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,11 +48,11 @@
...
     "rollup-plugin-vue": "^5.1.9",
     "vue": "^2.6.14",
-    "vue-template-compiler": "^2.6.14"
+    "vue-template-compiler": "^2.7.8"
   },
   "peerDependencies": {
     "@nextcloud/l10n": "^1.4",
-    "@nextcloud/vue": "^4.2||^5.0",
+    "@nextcloud/vue": "^6.0.0-beta.1",
     "vue": "^2.6"
```
Comment (juliushaertl) [inline package.json:56]: Needs a bump as well

**Comment by PVince81** [comment]
I've tried to test this by npm-linking it into https://github.com/nextcloud/server/pull/33508

and am getting this with `make watch-js`:

```
ERROR in ./node_modules/@nextcloud/calendar-availability-vue/node_modules/vue-material-design-icons/DotsHorizontal.vue 1:0
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> <template>
|   <span v-bind="$attrs"
|         :aria-hidden="!title"
 @ ./node_modules/@nextcloud/calendar-availability-vue/node_modules/@nextcloud/vue/dist/Components/DatetimePicker.js 2:369341-369396
 @ ./node_modules/@nextcloud/calendar-availability-vue/lib/index.esm.js 1:0-75 9:2-16
 @ ./node_modules/babel-loader/lib/index.js!./node_modules/vue-loader/lib/index.js??vue-loader-options!./apps/dav/src/views/Availability.vue?vue&type=script&lang=js& 43:0-76 58:26-46
 @ ./apps/dav/src/views/Availability.vue?vue&type=script&lang=js& 1:0-184 1:200-203 1:205-386 1:205-386
 @ ./apps/dav/src/views/Availability.vue 2:0-64 3:0-59 3:0-59 10:2-8
 @ ./apps/dav/src/settings-personal-availability.js 3:0-48 5:22-34
```

File: package.json (hunk 1 above)
Comment (PVince81) [reply package.json:56]: are you referring to updating vue ? I saw on server we also have vue 2.6, so probably need to move it up there as well

**Comment by skjnldsv** [comment]
> I've tried to test this by npm-linking it into [nextcloud/server#33508](https://github.com/nextcloud/server/pull/33508)

Most likely an issue with linking itself. Try manually copying the folder to `server/node_modules/@nextcloud/calendar-availability-vue`

**Review by juliushaertl — APPROVED** [review]
Tested and works

**Pull request merged**